config_converter/config_mapper/config_mapper.py

```diff
@@ -198,10 +202,11 @@ def _convert_in_tail(d: config_pb2.Directive, stats: dict) -> dict:
             stats['attributes_recognized'] += 1
         elif p.name in _UNSUPPORTED_FIELDS:
             stats['attributes_skipped'] += 1
-            print(f'{p.name} cannot be mapped into master agent config file.')
+            logging.warning(
+                '%s cannot be mapped into master agent config file', p.name)
```

Comment:
```suggestion
                'skip mapping field %s which is not a configurable parameter for master agent config file', p.name)
```

The motivation is to mindfully telling the users that we skip the field for a reason.
You can update the text suitable for the goal.

Reply:
Agreed - will change the text!